dev-libs/libxslt/libxslt-1.1.37-r1.ebuild

```diff
@@ -53,6 +53,8 @@ src_prepare() {
 }
 
 multilib_src_configure() {
+	append-lfs-flags
```

Comment:
Could you submit a PR upstream to add `AC_SYS_LARGEFILE` to configure.ac, then put it in a comment above this line?

Reply:
Thank you!